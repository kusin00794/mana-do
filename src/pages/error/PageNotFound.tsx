import React from 'react';
import { Link } from 'react-router-dom';

import '../../styles/common.scss';

const PageNotFound = () => {
  return (
    <div className="layout">
      <div className="row">
        <div className="col-12">
          <h1 className="text-center">Page Not Found</h1>
        </div>
        <div className="col-12">
          <div className="text-center">
            <Link to="/" className="text-decoration-none">
              <i className="fa fa-angle-left mr-2" />
              <span>Back to Home</span>
            </Link>
          </div>
        </div>
      </div>
    </div>
  );
};

export default PageNotFound;
