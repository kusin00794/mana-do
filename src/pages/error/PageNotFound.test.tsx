import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import { testSnapshots } from '../../utils/test';
import PageNotFound from './PageNotFound';

const Wrapper = (props: any) => (
  <BrowserRouter>
    <PageNotFound {...props} />
  </BrowserRouter>
);
describe('PageNotFound Page', () => {
  testSnapshots(Wrapper, [
    {
      props: {},
      description: 'render PageNotFound Page',
    },
  ]);
});
