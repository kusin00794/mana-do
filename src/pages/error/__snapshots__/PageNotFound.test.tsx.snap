// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Page snapshots render PageNotFound Page 1`] = `
<div
  className="layout"
>
  <div
    className="row"
  >
    <div
      className="col-12"
    >
      <h1
        className="text-center"
      >
        Page Not Found
      </h1>
    </div>
    <div
      className="col-12"
    >
      <div
        className="text-center"
      >
        <a
          className="text-decoration-none"
          href="/"
          onClick={[Function]}
        >
          <i
            className="fa fa-angle-left mr-2"
          />
          <span>
            Back to Home
          </span>
        </a>
      </div>
    </div>
  </div>
</div>
`;
